release.Jenkinsfile

```diff
@@ -0,0 +1,132 @@
+#!groovy
+
+def repo_owner = "pace-neutrons"
+def release_id_description = (
+  "The IDs of the jobs that contain the target release artifacts.\n" +
+  "This parameter should have the form:\n\n" +
+  "  <JOB_NAME> <BUILD_NUMBER>\n" +
+  "  <JOB_NAME> <BUILD_NUMBER>\n" +
+  "  ...")
+
+def get_agent() {
+  def agent_label = ''
+  withCredentials([string(credentialsId: 'win10_agent', variable: 'agent')]) {
+    agent_label = "${agent}"
+  }
+  return agent_label
+}
+
+def get_repo_name(String job_name) {
+  if (job_name.contains('Herbert/')) {
```

Comment:
`contains` or `startsWith` ?

Reply:
Contains. The full name would be e.g. `/PACE-neutrons/Herbert/Deploy`. Using `contains` seems the most straightforward way of doing this.